src/driver/transaction.rs

```diff
@@ -38,6 +39,7 @@ impl RustTransaction {
         rollback_savepoint: Arc<tokio::sync::RwLock<HashSet<String>>>,
         isolation_level: Option<IsolationLevel>,
         read_variant: Option<ReadVariant>,
+        deferable: Option<bool>,
```

Comment:
You need to add this parameter in `pyi` file for correct autosuggestions and type hints. You can find these fields in `python/psqlpy/_internal`